components/browser/state/src/test/java/mozilla/components/browser/state/action/TabListActionTest.kt

```diff
@@ -500,18 +501,18 @@ class TabListActionTest {
         store.dispatch(
             TabListAction.RestoreAction(
                 tabs = listOf(
-                    createTab(id = "c", url = "https://www.example.org", private = true),
-                    createTab(id = "d", url = "https://getpocket.com", private = false)
+                    RecoverableTab(id = "c", url = "https://www.example.org", private = true),
+                    RecoverableTab(id = "d", url = "https://getpocket.com", private = false)
                 ),
                 selectedTabId = "d"
             )
         ).joinBlocking()
 
         assertEquals(4, store.state.tabs.size)
-        assertEquals("c", store.state.tabs[0].id)
-        assertEquals("d", store.state.tabs[1].id)
-        assertEquals("a", store.state.tabs[2].id)
-        assertEquals("b", store.state.tabs[3].id)
+        assertEquals("a", store.state.tabs[0].id)
+        assertEquals("b", store.state.tabs[1].id)
+        assertEquals("c", store.state.tabs[2].id)
+        assertEquals("d", store.state.tabs[3].id)
```

Comment:
This is due to the refactor of adding tabs without restore indices to the end of the tab list instead of the prior behavior of adding them to the beginning.